MSAL/src/MSALPublicClientApplication.m

```diff
@@ -332,6 +297,88 @@ - (id)initWithClientId:(NSString *)clientId
                                    error:error];
 }
 
+#pragma mark - Keychain
+
+#if TARGET_OS_IPHONE
+- (BOOL)setuplTokenCacheWithConfiguration:(MSALPublicClientApplicationConfig *)config error:(NSError **)error
```

Comment:
Is there a typo "l" in the name of the method. setuplTokenCacheWithConfiguration.

Reply:
Addressed